stories/price.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Price } from '..';
+
+storiesOf('Prices', module).add('Price', () => (
+  <div>
+    <p>A regular Price for a currency with postpended symbol.</p>
```

Comment:
I don't think you need all the paragraphs :) Just the amounts are sufficient to see how they work. Otherwise, looks beautiful!

One question though - is there an option to set the color?